crates/ruff_python_ast/src/strings.rs

```diff
@@ -29,10 +36,37 @@ pub fn leading_quote(content: &str) -> Option<&str> {
     None
 }
 
-/// Return the trailing quote string for a docstring (e.g., `"""`).
+/// Return the trailing quote string for a string or byte literal (e.g., `"""`).
 pub fn trailing_quote(content: &str) -> Option<&&str> {
-    TRIPLE_QUOTE_SUFFIXES
+    ruff_python_stdlib::str::TRIPLE_QUOTE_SUFFIXES
         .iter()
-        .chain(SINGLE_QUOTE_SUFFIXES)
+        .chain(ruff_python_stdlib::str::SINGLE_QUOTE_SUFFIXES)
         .find(|&pattern| content.ends_with(pattern))
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_prefixes() {
+        let prefixes = ruff_python_stdlib::str::TRIPLE_QUOTE_PREFIXES
+            .iter()
+            .chain(ruff_python_stdlib::bytes::TRIPLE_QUOTE_PREFIXES)
+            .chain(ruff_python_stdlib::str::SINGLE_QUOTE_PREFIXES)
+            .chain(ruff_python_stdlib::bytes::SINGLE_QUOTE_PREFIXES)
+            .collect::<Vec<_>>();
+        for i in 1..prefixes.len() {
+            for j in 0..i - 1 {
```

Comment:
`..` ranges are exclusive, so this doesn't test all combinations (https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6eaf7a38144d10efe13d3455937b82ef). i'd just do `prefixes.enumerate()` twice here

Reply:
(Moved to #3396.)